llvm/test/Transforms/LoopInterchange/lcssa.ll

```diff
@@ -225,7 +225,7 @@ for.end16:                                        ; preds = %for.exit
 ; REMARK: UnsupportedExitPHI
 ; REMARK-NEXT: lcssa_06
 
-define void @lcssa_06(ptr %ptr, ptr %ptr1) {
+define void @lcssa_06(ptr %ptr, ptr noalias %ptr1) {
```

Comment:
Are all these other `noalias` changes required as part of the patch? If so, I can recommend marking both arguments `noalias`, and mentioning why in the commit message (eg. if these tests were faulty to begin with).